### DISCUSSION TITLE
Blank screen with bedrock setup

### DISCUSSION THREAD

**Discussion by sunilsingh2019** [wp-graphql/wp-graphql#2560] [General]
I am trying to install graphql with bedrock setup. I have found blank screen on graphql IDE.
Screenshots
![image](https://user-images.githubusercontent.com/26164995/193431362-aa20bd6b-ec62-45fe-9427-75a54b65eb36.png)
![image](https://user-images.githubusercontent.com/26164995/193431367-bddc2a8d-9800-43f2-91f3-46e006fb6acb.png)

Console error
Uncaught TypeError: $(...).wpColorPicker is not a function
    at HTMLDocument.<anonymous> (admin.php?page=graphql-settings:431:33)
    at mightThrow (jquery.js?ver=3.6.0:3766:29)
    at process (jquery.js?ver=3.6.0:3834:12)
companion-bubble.js:1452 Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at companion-bubble.js:1452:20487
    at Generator.next (<anonymous>)
    at An (companion-bubble.js:1452:19905)

**Comment by justlevine** [comment]
Uninstall WPGraphiQL it's now baked into the core plugin

**Reply by sunilsingh2019** [reply]
I have removed but still not working. my Url pattern is http://site-name.local/wp/graphql is that affect on graphql ?

**Reply by justlevine** [reply]
Try saving saving permalinks to regenerate the endpoint., And if you're still getting any errors in GraphiQL or the endpoint, share them
